### ISSUE TITLE
issue on windows 10

### ISSUE THREAD

**Issue by msprotz** [protz/ocaml-installer#41]
apparently the installer will not self-escalate to admin privileges on windows 10

**Comment by msprotz** [comment]
or, more precisely: it won't launch the cygwin setup it just downloaded

**Comment by markulf** [comment]
Also ran into this problem. What is the easiest workaround? Manually install cygwin and the required packages, which are those? See the `If you already have Cygwin64 installed` section on https://protz.github.io/ocaml-installer/.

**Comment by msprotz** [comment]
Yes, and also see https://github.com/FStarLang/FStar/blob/master/contrib/CoreCrypto/INSTALL.md for extra packages and more detailed instructions. If you have a patch I'll gladly take it, but I think we're both short on time :)
